src/main/java/org/llorllale/cactoos/matchers/Assertion.java

```diff
@@ -92,27 +97,24 @@
     public Assertion(
         final String msg, final T test, final Matcher<T> matcher
     ) {
-        this.refuted = new Unchecked<>(() -> !matcher.matches(test));
-        this.error = new Unchecked<>(
-            () -> {
-                final Description text = new StringDescription();
-                text.appendText(msg)
-                    .appendText(String.format("%nExpected: "))
-                    .appendDescriptionOf(matcher)
-                    .appendText(String.format("%n but was: "));
-                matcher.describeMismatch(test, text);
-                return new AssertionError(text.toString());
-            }
-        );
+        this.msg = msg;
```

Comment:
@victornoel Shouldn't the names of fields and ctor parameters be different?

Reply:
@andreoss some people seems to think it is important, but since I am ARC, I've never enforced that for anybody because I don't think it is useful or has any advantage, the opposite actually, it forces you to invent new names for the same thing...
Why do you think it should be the case? I could change my mind

Reply:
@victornoel I don't think this rule is important either. 
It was removed last year https://github.com/teamed/qulice/blame/e296098d1722f13ad6644f6a2958b5e8d1cbc47c/qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml#L201

Reply:
@andreoss good find!! thanks :+1: :)